Answers.md

```diff
@@ -2,10 +2,29 @@
 
 1. What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+React js is One or many ways to write large applications.It is an open source javascript library that is used to build user interfaces specifically.

```

Comment:
React JS is one of many Javascript libraries/frameworks that can be used to develop large scale front-end applications.